wirecard/oxid-ee
# WDPD-66: Live chat

### This PR

* Livechat added to the plugin settings page, payment gateway payment pages, transaction pages

### Notes

* As discussed, due to OXID eShop admin restrictions, we had to use static livechat window
* Chatting with the support team was tested, more info in ticket
* Fixes missing template issue

### How to Test

* Go to Admin
* Enable Payment Gateway plugin
* Red icon in the lower right corner should be rendered on the pages described above
* Clicking the icon should open the new livechat window

### Jira Links

* [WDPD-66](https://jira.parkside.at/browse/WDPD-66)
